client-sdk/ts-web/core/test/slip-0010-vectors.json

```diff
@@ -0,0 +1,82 @@
+[
+    {
+        "seed": "000102030405060708090a0b0c0d0e0f",
```

Comment:
This test vector set being in the library is somewhat questionable, though it isn't your fault.

The spec reads as thus:
> 1. Generate a seed byte sequence S of 512 bits according to BIP-0039.

I have no idea why the SLIP-0010 test vectors include results that are generated from invalid input.  The equivalent Go code in oasis-core strictly enforces that `S` is exactly 512 bits.  I would suggest that at a minimum, the user-callable API also does the same.

Reply:
It's for the BIP-0032 compatibility which allows 128 bits seeds - see [my response here](https://github.com/oasisprotocol/oasis-sdk/pull/119#discussion_r631505798)